### PR TITLE
Tournament model

### PR DESCRIPTION
**Pull request by compostbrain**

### CODE REVIEW

File: spec/factories/tournaments.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :tournament do
+    name "MyString"
```
Comment (alexstophel) [inline spec/factories/tournaments.rb]: I would suggest more realistic values.

File: db/migrate/20170918173132_create_tournaments.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreateTournaments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tournaments do |t|
+      t.string :name
```
Comment (alexstophel) [inline db/migrate/20170918173132_create_tournaments.rb]: Consider using `null: false` for columns that require data.

File: app/views/tournaments/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<%= form_for @tournament do |f| %>
+  <%= render "shared/errors", object: @tournament %>
+  <%= f.text_field :name%>
```
Comment (alexstophel) [inline app/views/tournaments/new.html.erb]: Add a space before `%>`

File: spec/factories/tournaments.rb (hunk 1 above)
Comment (compostbrain) [reply spec/factories/tournaments.rb]: fixed

File: db/migrate/20170918173132_create_tournaments.rb (hunk 2 above)
Comment (compostbrain) [reply db/migrate/20170918173132_create_tournaments.rb]: fixed

File: app/views/tournaments/new.html.erb (hunk 3 above)
Comment (compostbrain) [reply app/views/tournaments/new.html.erb]: fixed

**Pull request merged**